tutorial/docs/quickstart/dashboard-quickstart.mdx

```diff
@@ -0,0 +1,137 @@
+import JellyfishArchitecture from "./_jellyfish-architecture.mdx";
+import StartingJellyfishBackend from "./_starting-jellyfish-backend.mdx";
+import StartingJellyfishDashboard from "./_starting-dashboard.mdx";
+
+# The Whats and Hows of Jellyfish Dashboard
+
+## What is Jellyfish Dashboard?
+
+Jellyfish Dashboard is a web application that allows you to manage your
+Jellyfish backend. It is a React application that uses the Jellyfish React Client API
+to communicate with the backend.
+
+It was created to help Jellyfish developers in testing but was later extended and
+adapted to as a tool for tutorials and demos. Here we present a short introduction to
+the dashboard and its features.
+
+## How to start Jellyfish Dashboard?
+
+<StartingJellyfishDashboard />
+
+## How to start Jellyfish Backend?
+
+<StartingJellyfishBackend />
+
+## How to use Jellyfish Dashboard?
+
+Since we have Dashboard and Jellyfish backend up and running, we can dive into
+the features.
+
+### Connecting to the server
+
+We started the dashboard and the backend separately, so we need to connect them.
+
+1. Open the dashboard in your browser. Open the side menu and fill in the parameters:
+   ![Dashboard server connection](./img/dashboard_server.png)
+
+- **Server token** - a token necessary to authenticate the dashboard with the backend. For now, it's simply `development`, but
+  it is meant to authenticate the dashboard to the backend, so the good practice is to use a token that is hard to guess.
+- **WS/WSS** - whether to use a secure or insecure connection. , it's `ws`, but if the backend uses secure WebSocket, use `wss`.
+- **HTTP/HTTPS** - whether to use a secure or insecure connection. By default, it's `http`, but if the backend uses secure HTTP, use `https`.
+- **Server URL** - URL of the backend server. Here, `localhost:5002` would be a local build, but it can be any URL that points to the backend.
+- **Socket Path** - path to the WebSocket endpoint. The default value is `/socket/peer/websocket`.
+
+2. Click **Connect to server**. If everything is correct, a server will appear in the list of connected servers:
+   ![Connected dashboard](./img/dashboard_connected.png)
+
+### Creating a room
+
+There are plenty of settings that You can adjust in a room:
+![Adding Room](./img/dashboard_add_room.png)
+
+- **vp8 or h264** - video codec used in the room. If you want to use `HLS` streaming, you need to use `h264`.
```

Comment:
- You could also mention that you need a h264 to stream RTSP to WebRTC  
- on the screenshot h264 is the first value, so maybe change to`h264  or vp8` 🫨